addons/account/models/account_payment.py

```diff
@@ -400,12 +400,18 @@ def _compute_state(self):
             if not payment.state:
                 payment.state = 'draft'
             if (
-                not payment.outstanding_account_id
+                payment._has_no_outstanding()
```

Comment:
Ok, I got confused because I thought we needed the same condition as in `create` but actually not. So the function is useless, but then
```suggestion
                (not payment.outstanding_account_id or not accounting_installed)
```
or
```suggestion
                (not payment.move_id and payment.state == 'in_process')
```